Library/Formula/snag.rb

```diff
@@ -16,46 +16,19 @@ class Snag < Formula
 
   depends_on "go" => :build
 
-  go_resource "gopkg.in/yaml.v2" do
```

Comment:
I presume all these are vendored now if we're deleting them from the formulae? There's no external resource fetching during compile happening?